core/objects.go

```diff
@@ -692,7 +695,6 @@ type OCSPResponse struct {
 	// createdAt: The date the response was signed.
 	CreatedAt time.Time `db:"createdAt"`
 
-	// response: The encoded and signed CRL.
```

Comment:
Seems like a spurious deletion? I know we're going to remove this struct soon, but let's clear out this diff unless it's needed.